docs/sdk/1-workshop/02-overview.md

```diff
@@ -8,42 +8,47 @@ In this tutorial, we'll run through the full story of a claim.
 To do so, three actors will be involved: a <span class="label-role claimer">Claimer</span>, an <span class="label-role attester">Attester</span> and a <span class="label-role verifier">Verifier</span>.
 You'll be playing all three roles. In the real world, these actors would be running different services, so we set up different folders to mimic this separation.
 
-These three actors will be exchanging various objects.
-The most important one is the `credential`.
-This is how a `credential` is created:
+Both the <span class="label-role verifier">Verifier</span> and the <span class="label-role attester">Attester</span> have to interact with the KILT blockchain.
+But only the <span class="label-role attester">Attester</span> is required to own KILTs since he has to pay for storing attestation on chain.
+The <span class="label-role verifier">Verifier</span> only needs to query the KILT blockchain to ensure that the attestation is still valid and wasn't revoked.
+The <span class="label-role claimer">Claimer</span> is not required to query the blockchain, but he might do so to check whether their credential is still valid or the attester has revoked it.
+
+## Request an Attestation
+
+Before the <span class="label-role claimer">Claimer</span> can receive an attestation, he needs to generate a LightDID, which he can do completely off-chain.
```

Comment:
I would link to the definition of light DIDs as we cannot expect a user to know what this is.

```suggestion
Before the <span class="label-role claimer">Claimer</span> can receive an attestation, they need to generate a [light DID](../0-core-feature/0_did.md#light-dids), which they can do completely off-chain.
```